x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/parser/StatementParserTests.java

```diff
@@ -2939,4 +2939,14 @@ public void testNamedFunctionArgumentWithUnsupportedNamedParameterTypes() {
             );
         }
     }
+
+    public void testRemoteLookupJoin() {
+        expectError("FROM remote:left | LOOKUP JOIN right ON field", "LOOKUP JOIN does not support remote cluster indices [remote:left]");
```

Comment:
I think we should try and randomize the `remote:left`  as much as possible. In that position, we absolutely allow all kinds of stuff, including date math and wildcard patterns.